api/contrib/envoy/extensions/dlb/v3alpha/dlb.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+
+package envoy.extensions.dlb.v3alpha;
```

Comment:
I think this package (and file path) should be something more like extensions.connection_balance ?

Reply:
Yeah, it is more meaningful. Updated.